src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,47 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    public static final int CAPACITY = 10;
+
+    private Object[] keys;
+    private Object[] values;
```

Comment:
```suggestion
    private K[] keys;
    private V[] values;
```

That's where we should use class type parameters